src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultItems.props

```diff
@@ -1,8 +1,16 @@
-<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <PropertyGroup>
-    <EnableDefaultAndroidItems Condition=" '$(EnableDefaultAndroidItems)' == '' ">$(EnableDefaultItems)</EnableDefaultAndroidItems>
-  </PropertyGroup>
+<!--
+***********************************************************************************************
+Microsoft.Android.Sdk.DefaultItems.props
+WARNING:  This file is imported by AutoImport.props, and will be imported by all
+          projects using Microsoft.NET.Sdk.  All Item includes in this file *MUST* be
+          hidden behind an Android property based condition.
+
+Default item includes (globs and implicit references).
```

Comment:
Why do we want to define globs and implicit references here (where they'll be imported by all projects using Microsoft.NET.Sdks) as opposed to in a .targets file later (which would only be imported when building for Android), when they will be behind an Android-specific property anyway?

Reply:
This is the design here:

https://github.com/dotnet/designs/blob/main/accepted/2020/workloads/workload-resolvers.md#workload-props-files

I think it because the wildcards need to be defined before the body of the `.csproj` file. @dsplaisted might know more details.

@pjcollins we could maybe move the contents of file to `AutoImport.props`, maybe would be clearer that it needs to check for `$(TargetPlatformIdentifier)` everywhere?

Reply:
Yes, the default items need to be included before the body of the project file in order for VS to be able to override them.  IE if there is a `.cs` file and in VS you change the build action to `Content` in the VS UI, it will add something like the following to your project file:

```xml
<ItemGroup>
    <Compile Remove="ActuallyContent.cs" />
    <Content Include="ActuallyContent.cs" />
</ItemGroup>
```

For the `Remove` to work, the item needs to have been included beforehand, so the default items need to go in .props files before the body of the project.